docs/StardustDocs/topics/createDataFrame.md

```diff
@@ -151,8 +151,14 @@ df.add("length") { value.length }
 
 <!---END-->
 
-Creates a [`DataFrame`](DataFrame.md) from [`Iterable<T>`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/-iterable/) with one column:
+Creates a [`DataFrame`](DataFrame.md) from an [`Iterable<T>`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/-iterable/) with one column:
 "columnName: `DataColumn<T>`".
+This is an easy way to create a [`DataFrame`](DataFrame.md) when you have a list of Files, URLs, or a structure
+you want to extract data from.
+In a notebook,
+it can be convenient to start from the column of these values to see the number of rows, their `toString` in a table
+and then iteratively add columns with the parts of the data you're interested in.
+It could be File's content, a specific section of an HTML document, some metadata, etc.
```

Comment:
*a File's content :)

Reply:
since you're referring back to the list of files you started with. You could also write "a" if you don't want to refer

Reply:
Thank you c: